src/harness/sas_interface.py

```diff
@@ -404,4 +404,44 @@ def assertValidResponseFormatForApprovedGrant(self, grant_response):
       Nothing. It asserts if something about the response is broken/not per
       specs. Assumes it is dealing with an approved request.
     """
-    pass
\ No newline at end of file
+    pass
+
+  @abc.abstractmethod
+  def assertRegistered(self, registration_request,
+                       conditional_registration_data=None):
+    """Register a list of devices.
+
+    Quickly register N devices, assert registration SUCCESS, get CBSD IDs.
+    Includes injection of FCC IDs and conditional registration data.
+
+    Args:
```

Comment:
Can we pass a whole config file instead of individual arguments? Because if we are planning to use this with what has been proposed in https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/pull/228, it will be difficult to know when to use this assertion as the responseCodes are part of the Config File which has some 0s and other responseCode as well.